.zenodo.json

```diff
@@ -91,6 +91,9 @@
       "name": "Tirrell, Lee S.",
       "orcid": "0000-0001-9393-8361"
     },
+    {
+      "name": "TheJoggler"
```

Comment:
@TheJoggler Do you want to be an author on the Zenodo, and if so, what affiliation, name and [ORCID](http://orcid.org/) should we use?